tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/registered_cope_survey_suppression_test.py

```diff
@@ -0,0 +1,134 @@
+                f'{cls.project_id}.{cls.sandbox_id}.{sandbox_table_name}')
+
+        # call super to set up the client, create datasets
+        cls.up_class = super().setUpClass()
+
+    def setUp(self):
+        """
+        Create empty tables for the rule to run on
+        """
+        # Create domain tables required for the test
+        super().setUp()
+
+        # Load the test data
+        observation_data_template = self.jinja_env.from_string("""
+            INSERT INTO `{{project_id}}.{{dataset_id}}.observation`
+            (
+                observation_id, 
+                person_id, 
+                observation_concept_id, 
+                observation_type_concept_id,
+                value_as_concept_id,
+                observation_source_concept_id,
+                value_source_concept_id,
+                qualifier_concept_id,
+                unit_concept_id,
+                observation_date
+            )
+            VALUES
+            -- subset of Concepts to suppress --
+            -- 1310058 Thinking about your current social habits, in the last 5 days: -- 
+                    -- I have attended social gatherings of MORE than 50 people. Please specify: --
+            -- 1333325 In the past month, have you been sick for more than one day with a new illness related to -- 
+                    -- COVID-19 or flu-like symptoms? --
+            -- 1333235 Which of the following symptoms did you have? (select all that apply). --
+            -- 1310065 Which of the following symptoms did you have? Select all that apply. Please specify: --
+            -- 1333012 What breathing treatment did you receive? Please select all that apply. --
+            -- 1333234 What other breathing treatment did you receive? Please specify. --
+            -- 1332769 Who do you know who has died? Check all that apply. --
+            -- 702686  Who do you know who has died? Please specify. --
+            -- 1333156 Do you think you have had COVID-19? --
+            -- 713888  When did your COVID-19 symptoms begin? --
+              (1, 101, 0, 0, 0, 1310058, 0, 0, 0, '2020-01-01'),
+              (2, 102, 0, 0, 0, 1333325, 0, 0, 0, '2020-01-01'),
+              (3, 103, 0, 0, 0, 1333235, 0, 0, 0, '2020-01-01'),
+              (4, 104, 0, 0, 0, 1310065, 0, 0, 0, '2020-01-01'),
+              (5, 105, 0, 0, 0, 1333012, 0, 0, 0, '2020-01-01'),
+              (6, 106, 0, 0, 0, 1333234, 0, 0, 0, '2020-01-01'),
+              (7, 107, 0, 0, 0, 1332769, 0, 0, 0, '2020-01-01'),
+              (8, 108, 0, 0, 0, 702686, 0, 0, 0, '2020-01-01'),
+              (9, 109, 0, 0, 0, 715714, 0, 0, 0, '2020-01-01'),
+              (10, 110, 0, 0, 0, 1333156, 0, 0, 0, '2020-01-01'),
+              (11, 111, 0, 0, 0, 713888, 0, 0, 0, '2020-01-01'),
+              -- not concepts to be suppressed --
+              (12, 112, 0, 0, 0, 1111111, 0, 0, 0, '2020-01-01'),
+              (13, 113, 0, 0, 0, 2222222, 0, 0, 0, '2020-01-01')
```

Comment:
I would explicitly check the behaviour with null values too.  Since they sometimes behave unexpectedly.
```suggestion
              (13, 113, 0, 0, 0, 2222222, 0, 0, 0, '2020-01-01'),
              (14, 114, 0, 0, 0, null, 0, 0, 0, '2020-01-01')
```

Reply:
yeah, adding a null is a good idea, it didn't come to my thought while working. Thanks for the suggestion @lrwb-aou!!!